experimental/saiexperimentaldashvnet.h

```diff
@@ -84,11 +84,33 @@ typedef struct _sai_inbound_routing_entry_t
      */
     sai_object_id_t switch_id;
 
+    /**
+     * @brief Exact matched key eni_id
+     *
+     * @objects SAI_OBJECT_TYPE_ENI
+     */
+    sai_object_id_t eni_id;
+
     /**
      * @brief Exact matched key VNI
      */
     sai_uint32_t vni;
 
+    /**
+     * @brief Ternary matched key sip
+     */
+    sai_ip_address_t sip;
```

Comment:
should this be ip_prefix?

Reply:
It is a ternary match (priority based) with explicit priority. Prefix is used for LPM only.